### ISSUE TITLE
loss is NaN when using GCG on Llama-3.1-8B-Instruct (with default config)

### ISSUE THREAD

**Issue by songyangyifei** [facebookresearch/SecAlign#6]
Hi, sorry to bother you.
I'm running GCG attacks using the Llama-3.1-8B-Instruct model, but I consistently encounter loss = NaN during evaluation.

![Image](https://github.com/user-attachments/assets/65db69b8-e039-44d4-ad0c-2e4f3b72ea93)

**Comment by Sizhe-Chen** [comment]
Thank you for opening this issue and notifying me via email. We did not test Llama-3.1-8B-Instruct, but it is weird that it is not naturally supported. Does this error happen to other models we support?

I think to support new models, you should add its chat_template in config.py. That is, add 
"Llama-3.1-8B-Instruct": 
        ['<|begin_of_text|><|start_header_id|>system<|end_header_id|>', 
            '<|eot_id|><|start_header_id|>user<|end_header_id|>', 
            '<|eot_id|><|start_header_id|>assistant<|end_header_id|>'],

here https://github.com/facebookresearch/SecAlign/blob/main/config.py#L44
